### DISCUSSION TITLE
About the UI, well can we have some screencapts to start a tour of Ruined Fooocus ?

### DISCUSSION THREAD

**Discussion by AkoZ** [runew0lf/RuinedFooocus#82] [Ideas]
In fact i use Fooocus, and there are many good things, then i now evitate to install too fast an other SD software (updating about every ..day ?)
* the big matter is about **sharing the models** and all other embed..etc
* i suppose there is a .json or .txt **config file to put the pathes** but not sure.
* May be with easy exploration of the **screencaptures** one could know better what's inside the RuinedFooocus, coz even the log is here, the real way it is implemented is not visible.
@runew0lf **please, can you just screen-capt the parts of the RF** to know about the "advanced" parts and of course to compare better the UI, as your functions seem quite useful but some may be missing (gpt FooocusV2 style? or the in/outpainting ?), just the easy way is visual tour about your good work !

**Comment by runew0lf** [comment]
We've started work on the [Wiki](https://github.com/runew0lf/RuinedFooocus/wiki) where we are trying to add new features and document all the cool things you can do with RuinedFooocus.  The easiest way at the moment to get support or if you have any really pressing questions is to join our [Discord Server](https://discord.gg/CvpAFya9Rr)
